src/components/CompareResults/RevisionRow.tsx

```diff
@@ -39,6 +44,36 @@ const platformIcons: Record<PlatformShortName, ReactNode> = {
   Unspecified: '',
 };
 
+const getSubtestsCompareWithBaseLink = (result: CompareResultsItem) =>
+  `/subtestsCompareWithBase?baseRev=${result.base_rev}&
+baseRepo=${result.base_repository_name}&
+newRev=${result.new_rev}&
+newRepo=${result.new_repository_name}&
+framework=${result.framework_id}&
+baseParentSignature=${result.base_signature_id}&
+newParentSignature=${result.new_signature_id}`;
+
+const getSubtestsCompareOverTimeLink = (
+  result: CompareResultsItem,
+  interval: TimeRange['value'],
+) =>
+  `/subtestsCompareOverTime?baseRepo=${result.base_repository_name}&
+newRev=${result.new_rev}&
+newRepo=${result.new_repository_name}&
+framework=${result.framework_id}&
+interval=${interval}&
+baseParentSignature=${result.base_signature_id}&
+newParentSignature=${result.new_signature_id}`;
+
+const getSubtestsLink = (result: CompareResultsItem) => {
+  const { view } = useLoaderData() as LoaderReturnValue;
+  if (view == compareView) return getSubtestsCompareWithBaseLink(result);
+  else {
+    const { intervalValue } = useLoaderData() as LoaderReturnValue;
+    return getSubtestsCompareOverTimeLink(result, intervalValue);
+  }
+};
```

Comment:
I think I'd inline this function inside the component code directly:
```js
// Note that the return type is different depending on the view we're in
const loaderData = useLoaderData() as WithBaseLoaderReturnValue | OverTimeLoaderReturnValue;

const subtestsCompareLink = loaderData.view === compareView
  ? getSubtestsCompareWithBaseLink(result)
  : getSubtestsCompareOverTimeLink(result, loaderData.intervalValue);

// then use subtestsCompareLink below
```
What do you think? I think it's quite simple enough that it doesn't need an extra function.